core/src/main/java/de/kreggel/auth/core/api/Registration.java

```diff
@@ -1,4 +1,38 @@
 package de.kreggel.auth.core.api;
 
+import de.kreggel.auth.core.api.UserService;
+import de.kreggel.auth.core.persistence.jpa.entity.User;
+
+import javax.persistence.EntityManager;
+import javax.persistence.Persistence;
+import java.util.Date;
+
 public class Registration {
```

Comment:
Registration should add either a user as pending or have an extra table for registrations.